cmd/cloud/server.go

```diff
@@ -272,6 +277,63 @@ var serverCmd = &cobra.Command{
 	},
 }
 
+func healthCheck(awsConfig *sdkAWS.Config, s3StateStore string) error {
```

Comment:
Not gonna argue about the semantics, but maybe something like `checkRequirements` would fit this function a little better, what do you think?

Reply:
I agree, actually! I didn't put much thought into the name, as I was within the context of the ticket.